app/pods/publications/publication/case/template.hbs

```diff
@@ -112,21 +112,75 @@
               <h4 class="auk-panel__title">{{t "ministers-policy-fields"}}</h4>
             </WebComponents::AuToolbar::Item>
           </WebComponents::AuToolbar::Group>
-          <WebComponents::AuToolbar::Group @position="right">
-            <WebComponents::AuToolbar::Item>
-              {{!-- TODO check for the existence of latestSubcaseOnMeeting? --}}
-              {{!-- TODO scenario 4 --}}
-              {{!-- <WebComponents::AuButton @skin="tertiary" @icon="plus">{{t "add-minister"}}</WebComponents::AuButton> --}}
-            </WebComponents::AuToolbar::Item>
-          </WebComponents::AuToolbar::Group>
+          {{#if (not this.latestSubcaseOnMeeting)}}
+            <WebComponents::AuToolbar::Group @position="right">
+              <WebComponents::AuToolbar::Item>
+                <WebComponents::AuButton
+                  {{on "click" this.showMandateeModal}}
+                        @skin="tertiary"
+                        @icon="add"
+                        data-test-add-mandatee
+                >
+                  {{t "create-mandatee"}}
+                </WebComponents::AuButton>
+              </WebComponents::AuToolbar::Item>
+            </WebComponents::AuToolbar::Group>
+          {{/if}}
         </WebComponents::AuToolbar>
       </WebComponents::AuPanel::Header>
-      <WebComponents::AuPanel::Body @noPadding="true">
-        {{#if (not this.latestSubcaseOnMeeting.mandatees)}}
+      {{#if (not this.latestSubcaseOnMeeting)}}
+        {{# if (gt this.model.publicationFlow.mandatees.length 0) }}
+          <WebComponents::AuPanel::Body @noPadding="true">
```

Comment:
Via ministerraad lijkt er toch nog iets niet volledig correct te verlopen.

Als ik een nieuwe agenda maak met een nieuw dossier en een nieuwe procedurestap en ik agendeer deze op de nieuwe agenda, dan krijg ik nog de header `minister` te zien.. hier willen we eigenlijk de `emtpy state` lijkt me. 

Ik vermoed dat het hier dus nog ergens foutloopt?

![image](https://user-images.githubusercontent.com/11557630/106159231-db6f1680-6184-11eb-8829-bd4de69fba09.png)

Wanneer ik een minister toevoeg vanuit de agenda komt deze wel mooi bij in de publicatieflow.. Enkel de empty state toont daar nog niet correct.